ydb/docs/en/core/changelog-server.md

```diff
@@ -1,5 +1,85 @@
 # {{ ydb-short-name }} Server changelog
 
+## Version 25.1 {#25-1}
+
+### Version 25.1.1.18 {#25-1-1-18}
+
+Release date: 2025.
+
+#### Functionality
```

Comment:
❗Why is there no vector search stuff? If we're not releasing it yet, even as experimental under a feature flag, we'd better remove documentation about it from the `stable-25-1` branch.